src/api/senior/wishSeniorApply.ts

```
@@ -0,0 +1,29 @@
+import { ResponseModel } from '../model';
+
+export interface WishSeniorApplyRequest {
+  field: string;
+  postgradu: string;
+  professor: string;
+  lab: string;
+  phoneNumber: string;
+}
+
+interface WishSeniorApplyResponse extends ResponseModel {
+  data: {};
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)